mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/consumerstorage/KafkaConsumersStorage.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.verticle.consumers.consumerstorage;
+
+import org.folio.kafka.KafkaConsumerWrapper;
+
+import java.util.List;
+
+/**
+ * Storage for Kafka consumers. Consumers are not tenant specific.
+ */
+public interface KafkaConsumersStorage {
+
+  /**
+   * Adds new consumer.
+   * @param eventName the event name that is the key of consumer
+   * @param consumer  consumer wrapper to add
+   */
+  void addConsumer(String eventName, KafkaConsumerWrapper<String, String> consumer);
+
+  /**
+   * Gets consumer by event name.
+   *
+   * @param eventName the event name
+   * @return consumer wrapper by event name
+   */
+  List<KafkaConsumerWrapper<String, String>> getConsumersByEvent(String eventName);
```

Comment:
nitpick: these methods should return `Collection<KafkaconsumerWrapper<String,String>>` instead. This api is returning a List which means it is a collection that an item can be added or removed. Some people even return unmodifiable collection to reduce instance of bugs. Also returning an Array is cheaper than a List, especially since we dont need add/remove functionality of List.

Reply:
Modified return type to Collection instead of List